ltiauthenticator/lti11/validator.py

```diff
@@ -0,0 +1,89 @@
+import time
+
+from tornado import web
+
+from oauthlib.oauth1.rfc5849 import signature
+from collections import OrderedDict
+
+
+class LTILaunchValidator:
```

Comment:
LTI11LaunchValidator?

Reply:
@yuvipanda this is a class I created as an abstraction to validate the LTI 1.1 requests since some are required and some optional and wanted to offer end users the option of fine tuning their own validations. I would be glad to remove this though if it's noisy.

Reply:
Does 1.3 use the same validation logic? If so, then we can leave it as is! Otherwise just wanted a rename.

Thank you for going through this with me!

Reply:
It does, but we can rename it to something else if you'd like.

Reply:
Ah, I did not know that! If it's using the same validation logic, we can leave it as is. Thank you for the explanation :)